scripts/lib.sh

```diff
@@ -15,6 +15,14 @@ fi
 
 venv_name=".venv"
 
+# selected ansi colors
+TERM="${TERM:-xterm}"
+bold="$(tput bold)"
+red="$(tput setaf 1)"
+green="$(tput setaf 2)"
+yellow="$(tput setaf 3)"
+reset="$(tput sgr0)"
+
```

Comment:
there's code that sets this elsewhere and a comment as to why it cannot happen unconditionally